packages/mdc-slider/constants.js

```diff
@@ -35,6 +38,7 @@ const cssClasses = {
   TRACK_FILL: 'mdc-slider__track-fill',
   ACTIVE: 'mdc-slider--active',
   IN_TRANSIT: 'mdc-slider--in-transit',
+  IS_DISCRETE: 'mdc-slider--discrete',
```

Comment:
Rename this to just `DISCRETE`, since it's a string not a boolean

Reply:
Done!